packages/twenty-server/src/engine/api/graphql/graphql-query-runner/services/api-event-emitter.service.ts

```diff
@@ -5,6 +5,14 @@ import { ObjectMetadataInterface } from 'src/engine/metadata-modules/field-metad
 
 import { AuthContext } from 'src/engine/core-modules/auth/types/auth-context.type';
 import { WorkspaceEventEmitter } from 'src/engine/workspace-event-emitter/workspace-event-emitter';
+import { objectRecordChangedValues } from 'src/engine/core-modules/event-emitter/utils/object-record-changed-values';
+
+export enum DatabaseEventAction {
```

Comment:
as this is exported, this should go to it's own type file (under types subfolder)